### DISCUSSION TITLE
Build failed on my macOS 13.6.7

### DISCUSSION THREAD

**Discussion by mogocat** [ggml-org/llama.cpp#8288] [Q&A] [answered]
Just run the commands from readme page, but there is no main file to use. zsh: no such file or directory: ./main
M1 Max with macOS 13.6.7

**Comment by dspasyuk** [comment, marked as answer]
@mogocat it has been renamed to llama-cli:
../llama.cpp/llama-cli  --model ../../models/Meta-Llama-3-8B-Instruct_Q4_K_S.gguf  -cnv --ctx_size 0 --temp 0.3 --top_k 10 --chat-template llama3

**Reply by mogocat** [reply]
Why they didn't update the document?

**Reply by mogocat** [reply]
Thanks for your reply btw!
